mlsshared.sty

```diff
@@ -185,7 +188,7 @@
   breaklines=true,                 % automatic line breaking only at white-space
   keepspaces,                      % don't remove space such as those after closing parenthesis
   captionpos=b,                    % sets the caption-position to bottom
-  commentstyle=\color{commentcolor}\sffamily, % comment style
+  commentstyle=\color{commentcolor}, % comment style
```

Comment:
I thought the sans serif family was a carefully chosen style to make code comments look good at least in the pdfLaTeX build.  Is this a fix for the HTML build, or was there also a problem with the PDF?

Reply:
I have restored it with size to avoid that - but separately I think the sans serif family for PDF sometimes just looks bad; but we can resolve that separately.

Reply:
To be continued based on updated changes.